### PR TITLE
chore(ci): migrating to Ubuntu22 CI server

### PR DESCRIPTION
**Pull request by rdefosse**
fix(ci): some fields are now in hexadecimal or with a different hexa formatting

Also I am using the latest `2022.w44` RAN images, I had to change the process check.

Also adding some debug to help later maintenance.

### CODE REVIEW

**Comment by rdefosse** [comment]
I've tested this branch on:

* `orion`
* `cetautomatix`
* `selfix`

It's backward-compatible on `tshark` and `pyshark`

File: ci-scripts/rhel8SanityCheckReport.py [hunk 1]
```diff
@@ -342,28 +366,38 @@ def testSummaryDetails(self):
...
 				status = False
+				print ('SX Heartbeat => KO')
+			else:
+				print ('SX Heartbeat => OK')
 		if os.path.isfile('archives/ping_trf_gen_from_ue.log') or os.path.isfile('archives/ping_ue_from_trf_gen.log'):
 			details += self.addSectionRow('Ping Traffic Test')
 			res = self.checkPingLog('archives/ping_trf_gen_from_ue.log')
 			if res['status']:
-				print('Ping From UE : OK')
+				print('Ping From UE => OK')
```
Comment (arora-sagar) [inline ci-scripts/rhel8SanityCheckReport.py:385]: @rdefosse Changing from : to => won't affect the parsing in HTML at any place? Just asking to be sure

**Comment by arora-sagar** [comment]
@rdefosse Did you do a dry run with this branch ? Everything looks good on `selfix`?

File: ci-scripts/rhel8SanityCheckReport.py (hunk 1 above)
Comment (rdefosse) [reply ci-scripts/rhel8SanityCheckReport.py:385]: no. I've tested it.

**Comment by rdefosse** [comment]
> @rdefosse Did you do a dry run with this branch ? Everything looks good on `selfix`?

In progress: `https://jenkins-oai.eurecom.fr/blue/organizations/jenkins/MAGMA-MME-production/detail/MAGMA-MME-production/19943/pipeline/83`

**Comment by rdefosse** [comment]
Passing now: `https://jenkins-oai.eurecom.fr/view/MAGMA/job/MAGMA-MME-U18-Deploy-Check/1760/console`

**Comment by rdefosse** [comment]
@arora-sagar 

I will merge next week once we see there is no problem at all.

**Pull request merged**